certbot/main.py

```diff
@@ -405,6 +405,17 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _want_delete(config):
+    """Does the user want to delete their now-revoked certs?
+
+    """
+    prompt = (
+            'Would you like certbot to delete the revoked certificate '
+            'and related files on disk? Warning: your severs could be '
```

Comment:
I was echo-ing what was written in a comment on the original issue (without really thinking this through): https://github.com/certbot/certbot/issues/4109#issuecomment-274962861

My instinct is that it's better to warn the user just in case, but I guess if you're revoking your cert and then choosing to delete all files related to it, an extra warning does seem unnecessary. I'll modify this message to remove the warning.